proto/viam/app/data/v1/data.proto

```diff
@@ -229,6 +237,32 @@ message TabularDataByMQLResponse {
   reserved "data";
 }
 
+// ExportTabularDataRequest requests tabular data from the specified data source.
+message ExportTabularDataRequest {
+  string part_id = 1;
+  string resource_name = 2;
+  string resource_subtype = 3;
```

Comment:
[supernit] I think this should be resource type instead of subtype.

Reply:
Keeping it consistent with Katherine's API work. Also, Gloria pointed out: "ccording to the viam [glossary](https://docs.viam.com/appendix/glossary/#:~:text=type%20is%20component%20or%20service.%20subtype%20refers%20to%20a%20specific%20component%20or%20service%2C%20like%20a%20camera%20or%20vision.) resource type is either component or service and subtype is the actual component/service type like sensor or arm etc"